snapcraft.yaml

```diff
@@ -256,7 +256,7 @@ parts:
         [ "$(uname -m)" != "ppc64le" ] && exit 0
       set -ex
 
-      make USERCFLAGS=-Wno-error=format-truncation
+      make USERCFLAGS=-Wno-error=format-truncation criu
```

Comment:
what else was being built here?

Reply:
https://github.com/checkpoint-restore/criu/blob/criu-dev/Makefile#L167-L168:

```
# Default target
all: flog criu lib crit
```

At first, the `lib` one might seem interesting but looking at criu's prime section, the only 2 libs it primes are coming from the stage-packages `libnet1` and `libprotobuf-c1`.